data/transit/route/bus.json

```
@@ -18151,7 +18151,7 @@
         "network:wikidata": "Q3462238",
         "network:wikipedia": "en:Taichung City Bus",
         "network:zh": "臺中市市區公車",
-        "netwrok:wikipedia": "zh:臺中市市區公車",
+        "network:wikipedia:zh": "zh:臺中市市區公車",
```

I would just replace the English Wikipedia article with the Chinese one since this is the local language of this network
```suggestion
        "network:wikipedia": "zh:臺中市市區公車",
```